README.md

```diff
@@ -54,14 +50,24 @@ If the `@2x` version doesn't exist, the default icon will be used and scaled up
 
 Note that while this feature is supported for SVG icons, this is usually unneccesary because SVG, as a vector format, can be scaled without problems.
 
+### SDF icons support
+
+You can generate SDF icons by using the `--sdf` option. The generated sprite will be used in MapLibre GL JS with the `icon-color` and `icon-halo-color` properties.
```

Comment:
```suggestion
SDF icons can be generated by using the `--sdf` option. This allows users to apply [icon-color](https://maplibre.org/maplibre-style-spec/layers/#paint-symbol-icon-color) and [icon-halo-color](https://maplibre.org/maplibre-style-spec/layers/#paint-symbol-icon-halo-color) properties in Maplibre style to able to change the colour dynamically. 
```

changed some description here, and added maplibre style properties' links.

Reply:
Thank you, I resolve it.